frontend/src/container/SideNav/index.tsx

```diff
@@ -89,12 +89,14 @@ function SideNav(): JSX.Element {
 		},
 	];
 
+	const currentMenu = menus.find((menu) => pathname.startsWith(menu.to));
```

Comment:
@jyash97 can shift this one in helper,also can we wrap this one in useMemo?

Reply:
I dont think it would result in a significant performance improvement,since its just specific number of menus, and adding useMemo would add its own time complexeity

Reply:
@palashgdev do you still want this change ?

Reply:
no let it be